contracts/optimistic-ethereum/libraries/codec/Lib_OVMCodec.sol

```diff
@@ -37,6 +37,7 @@ library Lib_OVMCodec {
     }
 
     enum QueueOrigin {
+        UNUSED, // Save gas by not setting a storage slot to zero
```

Comment:
This one actually has the chance of requiring downstream changes due to how this enum is handled in geth.  I think we may want to take the efficiency loss to keep this as it was.  CC @karlfloersch  do you know?

Reply:
Talked to the team, def not worth the risk of missing an external dependency -- let's revert this one.